strelets/adapter/orchestrator.go

```diff
@@ -64,6 +65,7 @@ type Orchestrator interface {
 
 type OrchestratorOptions struct {
 	StorageDriver          string            `json:"storage-driver"`
+	StorageMountType string `json:"storage-mount-type"`
```

Comment:
fields `StorageDriver` and `StorageMountType` are redundant. instead of adding a field, type, getter and condition-  just add a new enum value to `StorageDriver` indicating local/bind (as opposed to local/volume, which is the current default)

Reply:
I think that `StorageDriver`, `StorageOptions` and `StorageMountType` will all go away when we get rid of EBS.